rest-api/pom.xml

```diff
@@ -101,7 +101,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-json-org</artifactId>
-            <version>2.3.3</version>
+            <version>${jackson.version}</version>
```

Comment:
Can we not just remove these versions? As the version within the dependency management section in the parent pom should be automatically used.